lib/parser.js

```diff
@@ -49,6 +55,31 @@ function parseMetadataComments(valuesFilePath, config) {
       currentSection = sectionMatch[1];
     }
 
+    // Parse section description end line
+    const sectionDescEndMatch = line.match(sectionDescriptionEndRegex);
+    if (currentSection && sectionDescriptionParsing && sectionDescEndMatch) {
+      sectionDescriptionParsing = false;
+      parsedSectionDescriptions[currentSection] = currentSectionDescriptionContent;
+      currentSectionDescriptionContent = [];
+    }
+
+    // Parse section description content line between start and end
+    if (currentSection && sectionDescriptionParsing) {
+      const sectionDescContentMatch = line.match(sectionDescriptionContentRegex);
+      if (sectionDescContentMatch && sectionDescContentMatch.length > 0) {
+        currentSectionDescriptionContent.push(sectionDescContentMatch[1]);
+      }
+    }
+
+    // Parse section description start line
+    const sectionDescStartMatch = line.match(sectionDescriptionStartRegex);
+    if (currentSection && sectionDescStartMatch) {
+      sectionDescriptionParsing = true;
+      if (sectionDescStartMatch.length > 0 && sectionDescStartMatch[1] !== '') {
+        currentSectionDescriptionContent.push(sectionDescStartMatch[1]);
+      }
+    }
+
```

Comment:
Following the suggested approach about `section` class, this could be just creating new section objects when finding the initial tag, and pushing lines to the description field until we find the closing tag.